packages/core/addon/components/visualization-config/wrapper.js

```diff
@@ -11,10 +11,67 @@
  */
 
 import Component from '@ember/component';
+import { get, set, computed } from '@ember/object';
+import { getOwner } from '@ember/application';
 import layout from '../../templates/components/visualization-config/wrapper';
+import { getRequestMetrics } from 'navi-core/utils/chart-data';
+import { copy } from 'ember-copy';
 
 export default Component.extend({
   layout,
 
-  classNames: ['visualization-config']
+  classNames: ['visualization-config'],
+
+  /**
+   * @property {Object} series
+   */
+  series: computed('visualization.metadata', function() {
+    return get(this, 'visualization.metadata.axis.y.series');
+  }),
+
+  /**
+   * @property {Object} seriesConfig
+   */
+  seriesConfig: computed('series', function() {
+    return get(this, 'series.config');
+  }),
+
+  /**
+   * @property {Boolean} showMetricSelect - whether to display the metric select
+   */
+  showMetricSelect: computed('visualization', 'series', function() {
+    const series = get(this, 'series'),
+      { visualization: { type }, request } = this,
```

Comment:
make sure this works in ember 2.x

Reply:
Thanks for the reminder to test in 2.x 🙏 
This works since both `visualization` and `request` are not cp's, but `this.metricName` had to be changed `this.get('metricName')`